src/NzbDrone.Core/Download/TrackedDownloads/TrackedDownloadService.cs

```diff
@@ -57,23 +58,35 @@ public TrackedDownload TrackDownload(DownloadClientDefinition downloadClient, Do
             try
             {
                 var parsedEpisodeInfo = Parser.Parser.ParseTitle(trackedDownload.DownloadItem.Title);
-                if (parsedEpisodeInfo == null) return null;
+                var historyItems = _historyService.FindByDownloadId(downloadItem.DownloadId);
 
-                var remoteEpisode = _parsingService.Map(parsedEpisodeInfo);
+                if (parsedEpisodeInfo != null)
+                {
+                    trackedDownload.RemoteEpisode = _parsingService.Map(parsedEpisodeInfo);
+                }
 
-                if (remoteEpisode.Series == null || !remoteEpisode.Episodes.Any())
+                if (historyItems.Any())
                 {
-                    var historyItems = _historyService.FindByDownloadId(downloadItem.DownloadId);
+                    var firstHistoryItem = historyItems.OrderByDescending(h => h.Date).First();
+                    trackedDownload.State = GetStateFromHistory(firstHistoryItem.EventType);
 
-                    if (historyItems.Empty())
+                    if (parsedEpisodeInfo == null ||
+                    trackedDownload.RemoteEpisode == null ||
```

Comment:
Align all the clauses of the if statement (makes it a bit more clear they're part of the same conditional):

```
if (statement1 = 1 ||
    statment2 = 2 ||
    etc...)
{
// Do something
}
```